docs/sources/reference/builder.md

```diff
@@ -782,9 +782,21 @@ and mark it as holding externally mounted volumes from native host or other
 containers. The value can be a JSON array, `VOLUME ["/var/log/"]`, or a plain
 string with multiple arguments, such as `VOLUME /var/log` or `VOLUME /var/log
 /var/db`.  For more information/examples and mounting instructions via the
-Docker client, refer to [*Share Directories via Volumes*](/userguide/dockervolumes/#volume)
+Docker client, refer to 
```

Comment:
"Readers use technical documents to perform tasks or gather information. For readers, these activities take place in the present. Therefore, the present tense is appropriate in most cases."   --->  which is why I'm editing this.  All that future tense with passive voice is resulting in what will be very difficult to parse sentence structures. :-D  Not your fault but I'm trying to correct as we go.

---

## VOLUME

    VOLUME ["/data"]

The `VOLUME` instruction creates a mount point with the specified name
and marks it as holding externally mounted volumes from native host or other
containers. The value can be a JSON array, `VOLUME ["/var/log/"]`, or a plain
string with multiple arguments, such as `VOLUME /var/log` or `VOLUME /var/log
/var/db`.  For more information/examples and mounting instructions via the
Docker client, refer to 
[*Share Directories via Volumes*](/userguide/dockervolumes/#volume)
documentation.

The `docker run` command initializes the newly created volume with any data that
exists at the specified location within the base image. For example, consider the
following Dockerfile snippet:

    FROM ubuntu
    RUN mkdir /myvol
    RUN echo "hello world" > /myvol/greating
    VOLUME /myvol

This Dockerfile results in an image that causes `docker run`, to
create a new mount point at `/myvol` and copy the  `greating` file 
into the newly created volume.

> **Note**:
> The list is parsed as a JSON array, which means that
> you must use double-quotes (") around words not single-quotes (').